JapaneseFilter/sections/whitelist.txt

```diff
@@ -7,6 +7,8 @@
 @@||accesstrade.net/m/m_img/$image,domain=mamastar.jp
 ! https://github.com/AdguardTeam/AdguardFilters/issues/79181
 @@||dospara.co.jp/banner/
+! https://github.com/AdguardTeam/AdguardFilters/issues/80265
+@@||log.nordot.jp/js/beacon-*.js$domain=kiji.is
```

Comment:
This request/script is blocked by AdGuard Tracking Protection filter, so I think that it would be better to add this rule to that filter list.